verticapy/jupyter/extensions/chart_magic.py

```diff
@@ -192,30 +192,427 @@ def chart_magic(
+        if  you  want to execute the  input
+        file.
+    -k / --kind  : str, optional
+        Chart Type, one  of  the following:
+
+        **area**         :
+                            Area Chart
+
+        **area_range**   :
+                            Area Range Chart
+
+        **area_ts**      :
+                            Area Chart with Time Series Design
+
+        **bar**          :
+                            Bar Chart
+
+        **biserial**     :
+                            Biserial Point Matrix (Correlation
+                            between binary variables and numerical)
+
+        **boxplot**      :
+                            Box Plot
+
+        **bubble**       :
+                            Bubble Plot
+
+        **candlestick**  :
+                            Candlestick and Volumes (Time Series
+                            Special Plot)
+
+        **cramer**       :
+                            Cramer's V Matrix (Correlation between
+                            categories)
+
+        **donut**        :
+                            Donut Chart
+
+        **donut3d**      :
+                            3D Donut Chart
+
+        **heatmap**      :
+                            Heatmap
+
+        **hist**         :
+                            Histogram
+
+        **kendall**      :
+                            Kendall Correlation Matrix
+
+        .. Warning::
+            This method  uses a CROSS JOIN  during  computation and is
+            therefore computationally expensive at  O(n * n),  where n
+            is the  total  count of  the ``vDataFrame``.
+
+        **line**         :
+                            Line Plot
+
+        **negative_bar** :
+                            Multi-Bar Chart for binary classes
+
+        **pearson**      :
+                            Pearson Correlation Matrix
+
+        **pie**          :
+                            Pie Chart
+
+        **pie_half**     :
+                            Half Pie Chart
+
+        **pie3d**        :
+                            3D Pie Chart
+
+        **scatter**      :
+                            Scatter Plot
+
+        **spider**       :
+                            Spider Chart
+
+        **spline**       :
+                            Spline Plot
+
+        **stacked_bar**  :
+                            Stacker Bar Chart
+
+        **stacked_hist** :
+                            Stacked Histogram
+
+        **spearman**     :
+                            Spearman Correlation Matrix
+
+    -o / --output : str, optional
+        Output File. You can use this option
+        if  you want to export the result of
+        the query to the HTML format.
+
+    Returns
+    -------
+    Chart Object
+
+    Examples
+    --------
+    The following examples demonstrate:
+
+    * Setting up the environment
+    * Drawing graphics
+    * Exporting to HTML
+    * Using variables
+    * Using SQL files
+
```

Comment:
To see more examples, please refer to the Chart Guide.

@mail4umar can you please use the right ref for the above sentence?

Reply:
Here it is: @tconsigny 

To see more examples, please refer to the ref:`chart_gallery.guide`.